lib/models/User.dart

```diff
@@ -0,0 +1,42 @@
+class User {
+  String id;
+  String age;
+  String city;
+  String firstName;
+  String lastName;
+  String gender;
+  String profession;
+  String email;
+  String username;
+  String password;
+  String createdAt;
+  String deletedAt;
+
+  User(
+      this.age,
+      this.city,
+      this.createdAt,
+      this.deletedAt,
+      this.email,
+      this.firstName,
+      this.gender,
+      this.lastName,
+      this.password,
+      this.profession,
+      this.username) {}
```

Comment:
Constructors should be at the top of the class. Can you move this to line 2 and the variables declarations below the constructor